mentorship ios/Registration/LoginView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginView.swift
+//  mentorship ios
+//
+//  Created by Yugantar Jain on 01/06/20.
```

Comment:
@isabelcosta @mayburgos what should be here?

Reply:
That line does not need to be there in my opinion. We even taken that out on Android app I think. these files are live, changed but the community so for me it does not make sense to have the file creator, since this appears on git @annabauza

Reply:
There is also a copyright line inserted by xcode @annabauza @isabelcosta 
" //  Copyright © 2020 Yugantar Jain. All rights reserved."

Reply:
then it could also be removed, right?

Reply:
yes!